website/content/en/preview/concepts/nodeclasses.md

```diff
@@ -431,6 +431,8 @@ spec:
 
 AMI Selector Terms are used to configure custom AMIs for Karpenter to use, where the AMIs are discovered through ids, owners, name, and [tags](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/Using_Tags.html). **When you specify `amiSelectorTerms`, you fully override the default AMIs that are selected on by your EC2NodeClass [`amiFamily`]({{< ref "#specamifamily" >}}).**
 
+Note that [`amiFamily`]({{< ref "#specamifamily" >}}) determines the bootstrapping mode, while amiSelectorTerms specifies specific AMIs to be used. Therefore, you need to ensure consistency between [`amiFamily`]({{< ref "#specamifamily" >}}) and amiSelectorTerms to avoid conflicts during bootstrapping.
```

Comment:
```suggestion
{{% alert title="Note" color="primary" %}}
[`amiFamily`]({{< ref "#specamifamily" >}}) determines the bootstrapping mode, while `amiSelectorTerms` specifies specific AMIs to be used. Therefore, you need to ensure consistency between [`amiFamily`]({{< ref "#specamifamily" >}}) and `amiSelectorTerms` to avoid conflicts during bootstrapping.
{{% /alert %}}
```

Can you also make this change to following folders? 
- /docs
- /v0.32
- /v0.34
- /v0.35
- /v0.36

Reply:
Of course :)

Reply:
Are you also planning on making the changes to the other files?